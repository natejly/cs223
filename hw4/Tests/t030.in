P1,3,1,1,3
P2,1,3,3,1

P1 P2
P2 P1
