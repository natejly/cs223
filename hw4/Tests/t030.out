P1 5.0 - P2 5.0
P2 5.0 - P1 5.0
